client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.tsx

```diff
@@ -50,20 +50,18 @@ function TierUpgradeSlider() {
 	// Get the plan details.
 	const plans = getPlanTiers();
 
-	// Adjust available plans based on current tier.
-	const availablePlans = plans;
-
 	// Slider state.
 	const [ currentPlanIndex, setCurrentPlanIndex ] = useState( 0 );
 	const sliderMin = 0;
 	const sliderMax = plans.length - 1;
 
 	const handleSliderChange = ( value: number ) => {
-		// console.log( 'slider did change. value: ', value );
-		// console.log( 'plan info: ', plans[value] );
 		setCurrentPlanIndex( value );
 	};
 
+	// Create an array of indices for the marks
+	const marks = plans.map( ( _, index ) => index );
```

Comment:
This shouldn't be necessary since we want marks for each item in the array. Should be enough to pass `true` to the slider.